README.md

```diff
@@ -39,17 +39,16 @@ First, make sure that you have the following dependencies:
 [Node]: https://nodejs.org/en/
 [Yarn]: https://yarnpkg.com/lang/en/
 [GitHub API token]: https://github.com/settings/tokens
-
 [macos-gnu]: https://github.com/sourcecred/sourcecred/issues/698#issuecomment-417202213
 
-Then, run the following commands to clone and build SourceCred:
+You'll stil need to create a GitHub token to use as an environment variable
```

Comment:
Creating a GitHub token is already discussed in the bullet list above; is there a reason we are mentioning it again?

Reply:
Redundancy is a good strategy for documentation. It's likely that a user sees one or the other, skips over things.